Gordon360/ApiControllers/HousingController.cs

```diff
@@ -79,6 +80,25 @@ public IHttpActionResult RemoveHousingAdmin(string id)
             return Ok(result);
         }
 
+        /// <summary>
+        /// Get a list of the apartment-style halls
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("halls")]
```

Comment:
you might want to call is "apartments/halls" or probably "halls/apartments", since the apartments are a subset of the set of all halls

Reply:
```suggestion
        [Route("halls/apartments")]
```

Reply:
Thanks, I changed it